File: Rust/src/String/is_string_whitespaces_only.rs
#![allow(clippy::needless_return)]
fn main() {
    // Check if a given string is whitespaces only.
    println!("{}", is_string_whitespaces_only("     "));
}

// Check if a string is whitespaces only.
fn is_string_whitespaces_only(content: &str) -> bool {
    return content.trim().is_empty();
}

File: TypeScript/String/is-string-whitespaces-only.ts
function main(): void {
    // Check if a given string is whitespaces only.
    console.log(isStringWhiteSpacesOnly("           "))
}

main()

// Check if a given string is whitespaces only.
function isStringWhiteSpacesOnly(content: string): boolean {
    return content.trim().length == 0
}
